packages/Pagination/Panels.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import safeRest from '@tds/shared-safe-rest'
+import hash from './hash'
+
+const Panels = ({ children, ...rest }) => (
```

Comment:
It looks like `<Panels>` only ever receives a [single child](https://github.com/telus/tds-community/pull/195/files#diff-3d68a15f8573d428bab5699d54b49a58R138). Is there a reason why it needs loop through `children`?